bundled_scripts/ilo.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import os
+import sys
+from copy import deepcopy
+
+import hpilo
+
+
+USER = os.environ['MANAGEMENT_USER_NAME']
```

Comment:
imo we should unify somehow these parameters. some are passed as a script param (ex. IP address), some are taken from env variables. Maybe we could use sth similar to this: http://stackoverflow.com/a/10551190/761782 - then we could specify variable either by env variable or by script param?